src/components/CulturePage/TeamCardList.tsx

```diff
@@ -0,0 +1,125 @@
+import styled from '@emotion/styled';
+import type { FC } from 'react';
+
+import { atMinDesktop, atMinTablet, atMinXL, spacing } from '~/theme';
+
+import { TeamMemberCard } from './TeamMemberCard';
+
+const TeamMasonry = styled.section`
+  --column-shift: ${spacing.xxl};
+  display: grid;
+  grid-template-columns: 1fr;
+  margin-bottom: 11rem;
+  position: relative;
+
+  img {
+    object-fit: cover;
+    z-index: 2;
+  }
+
+  ${atMinTablet} {
+    --column-shift: ${spacing.xxl3};
+    gap: 2.875rem;
+    grid-template-columns: repeat(2, 1fr);
+    margin-bottom: 15rem;
+
+    & article {
+      break-inside: avoid;
+      margin-bottom: 4rem;
+    }
+  }
+
+  ${atMinXL} {
+    grid-template-columns: repeat(3, 1fr);
+
+    & article {
+      margin-bottom: 0;
+      margin-top: 8rem;
+      width: 100%;
+      z-index: 2;
+    }
+    > * {
```

Comment:
💭 Maybe a newline here to stay consistent to the formatting between the other selector groups?

```suggestion
    }
    
    > * {
```